sr_utilities_common/src/sr_utilities_common/aws_manager.py

```diff
@@ -33,6 +33,7 @@ def __init__(self, access_key=None, secret_key=None, session_token=None):
         aws_access_key_id = access_key
         aws_secret_access_key = secret_key
         aws_session_token = session_token
+        retries = 5
```

Comment:
should this be a class constant? If not, perhaps parametrising it may be useful

Reply:
done